### DISCUSSION TITLE
Examples for TCA

### DISCUSSION THREAD

**Discussion by bitsarat** [pointfreeco/swift-composable-architecture#2798] [Q&A] [answered]
Hi @PointFree, 
Thanks for the livestream yesterday. I am waiting for @Shared already!
Can you please let us know what are the differences in between the example apps you have provided? What would we learn from each of them? I know Standups is the main one you discuss in the sessions.

**Comment by mbrandonw** [comment]
Hi @bitsarat, I'm sorry but I'm not sure I understand the question. Which examples are you talking about, and what kind of differences are you curious about?

**Reply by bitsarat** [reply]
I mean, what will TicTacToe teach me that StandUps will not. What will I get from ToDos? I am trying to navigate the examples you provide.

**Reply by bitsarat** [reply, marked as answer]
@mbrandonw Never mind. I went inside the writeups for each, and figured it out. Sorry for not doing it earlier!
